apps/oc_chef_wm/src/oc_chef_wm_key_util.erl

```diff
@@ -0,0 +1,67 @@
+%% -*- erlang-indent-level: 4;indent-tabs-mode: nil; fill-column: 80 -*-
```

Comment:
I like the shared behavior, but any reason not to use oc_chef_wm_keys:auth_info instead of adding the new module?